### PR TITLE
Update bo.m3u

### PR DESCRIPTION
**Pull request by sguinetti**
This PR adds Red Uno and CTV

### CODE REVIEW

**Comment by heroku-miraheze** [comment]
@sguinetti Why you don't make a review for my PR #5325???

**Review by heroku-miraheze — APPROVED** [review]

**Comment by sguinetti** [comment]
> @sguinetti Why you don't make a review for my PR #5325???

Sorry, I am not the owner of the repository and I am busy with other matters. However, I note you that I can't access the Pluto streams for now in VLC (because of USA), so I will try to ask the next colleague for help.

**Review by BellezaEmporium — DISMISSED** [review]

**Comment by freearhey** [comment]
@heroku-miraheze I warned you.

File: channels/bo.m3u [hunk 1]
```diff
@@ -5,16 +5,22 @@ https://593b04c4c5670.streamlock.net/8192/8192/playlist.m3u8
...
 https://mediacp.hostradios.com.ar:19360/atbcochabamba/atbcochabamba.m3u8
-#EXTINF:-1 tvg-id="BoliviaTV.bo" tvg-country="BO" tvg-language="Spanish" tvg-logo="http://www.boliviatv.bo/principal/assets/images/the-voyager/btv1.png" group-title="",Bolivia TV (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="BoliviaTV.bo" tvg-country="BO" tvg-language="Spanish" tvg-logo="http://www.boliviatv.bo/principal/assets/images/the-voyager/btv1.png" group-title="General",Bolivia TV (720p) [Not 24/7]
 http://boliviatv1.srfms.com:5735/live/livestream/playlist.m3u8
-#EXTINF:-1 tvg-id="BoliviaTV72.bo" tvg-country="BO" tvg-language="Spanish" tvg-logo="https://i.imgur.com/1zTLGdj.jpg" group-title="",Bolivia TV 7.2 (720p)
+#EXTINF:-1 tvg-id="BoliviaTV72.bo" tvg-country="BO" tvg-language="Spanish" tvg-logo="https://i.imgur.com/1zTLGdj.jpg" group-title="General",Bolivia TV 7.2 (720p)
 https://video1.getstreamhosting.com:1936/8224/8224/playlist.m3u8
+#EXTINF:-1 tvg-id="CTV.bo" tvg-country="BO" tvg-language="Spanish" tvg-logo="" group-title="Local",CTV Canal 38 Digital (Viacha)
+https://media2.cdnlayer.biz:8081/8026/index.m3u8
```
Comment (freearhey) [inline channels/bo.m3u:13]: image

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked link again.

File: channels/bo.m3u (hunk 1 above)
Comment (sguinetti) [reply channels/bo.m3u:13]: Tagged as Not 24/7

**Comment by LaneSh4d0w** [comment]
Will check back when I can confirm channel is online.

**Review by BellezaEmporium — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**